%---------------------------------------------------------------------
%
%                      resumen.tex
%
%---------------------------------------------------------------------
%
% Contiene el capítulo del resumen.En Inglés
%
% Se crea como un capítulo sin numeración.
%
%---------------------------------------------------------------------

\chapter{Summary}
\cabeceraEspecial{Summary}

%\begin{FraseCelebre}
%\begin{Frase}

%\end{Frase}
%\begin{Fuente}

%\end{Fuente}
%\end{FraseCelebre}

\subsubsection*{Introduction}
The concept of Proteomics, a term coined in analogy to Genomics,
was first used by Marc Wilkins in the mid 90s to describe
the total set of proteins being expressed by the genes of a cell,
tissue or organism.
Earlier, in the late 80s, the development of soft ionizations techniques, 
such as Electrospray Ionization (ESI) and Soft Laser Desorption (SLD)
enabled the ability to ionize large bio-molecules such as proteins while
keeping them relatively intact. This settled the foundation upon which
Mass Spectrometry was applied to what would later become modern Proteomics.

In \textit{shotgun} Proteomics, the first step of the experiment usually
consists of a digestion of the sample proteins into peptides by means of
a proteolytic enzyme such as trypsin. This greatly increases performance
in terms of the number of proteins that can be detected in a single 
LC-MS/MS run compared to gel-based approaches, but comes at the cost 
of a great complexity at the peptide level and the protein inference problem. 

Peptides are then separated by liquid chromatography (LC), then ionized
and eventually enter the mass spectrometer where they are separated
as a function of their mass-to-charge (\mz) ratio, recorded in the MS1 spectrum.
In tandem mass spectrometry (MS/MS)
peptides with higher intensities are selected to be fragmented so 
MS/MS spectra, a collection of \mz values and intensities of the precursor and product ions, are produced.

Once the empirical spectra are acquired, the computational analysis starts.
The most efficient peptide identification method is based on searching 
the acquired spectra against protein sequence databases. This is what
search engines do. Basically, a score measuring the degree of similarity 
between the empirical spectrum and a theoretically derived spectrum (corresponding to a known 
sequence) is given to pairs of spectrum - peptide sequence named PSMs (Peptide to Spectrum Match)

To assess confidence in peptide identification statistic measures such
as p-values and e-values are given to PSMs. But in the context of a large
experiment generating thousands of MS/MS spectra, further filtering or additional
statistical procedures may be applied.

The False Discovery Rate
Probability PeptideProphet

Then there is the protein inference problem: 2 problems
addressed by ProteinProphet non-random peptide-to-protein grouping which explodes FDR levels
and peptide degeneracy, that is, conserved sequences in different proteins


\subsubsection*{Objectives}
The presence of proteomic data related to the fungal opportunistic 
pathogen \textit{Candida albicans} on online public repositories was
until recently very sparse, sometimes originated from low resolution
instruments and therfore not very robust.
In this context, the development databases and the adoption
of standard formats in Proteomics have an essential role in the analysis,
sharing and dissemination of results. The software tools presented here 
contribute to these objectives.

\subsubsection*{Results}
The database and web tool Proteopathogen was the first described software 
that combined Proteomics experiments results with specific information 
relevant to the study of \textit{C.albicans} proteins such as GO (Gene Ontology)
terms and KEGG (Kyoto Encyclopedia of Genes and Genomes) pathways annotations.
Following a first version where the results were collected
in a tab separated text format dependant on the software used to generate 
them \citep{Vialas2009b}, the software was adapted to make use of
the standard format for protein and peptide identifications, mzIdentML, as its source of data
\citep{Vialas2015}. Proteopathogen is up to date an online tool to allow
visualization and analysis of Proteomics experiments results by the wet-lab 
users but also enabling revision by reviewers of the journals of the field.

At the same time, the development of a \textit{Candida albicans} PeptideAtlas
means the most exhaustive proteome characterization up to the current date.
It describes over 71000 detected peptides assigned to 4174 proteins, which
represent 66\% of the predicted proteome. Is is, in addition, the first 
model of a fungal pathogen present in the global PeptideAtlas project and
benefits from the robustness and confidence that TPP (Trans Proteomics Pipeline)
enables.

\endinput
% Variable local para emacs, para  que encuentre el fichero maestro de
% compilación y funcionen mejor algunas teclas rápidas de AucTeX
%%%
%%% Local Variables:
%%% mode: latex
%%% TeX-master: "../Tesis.tex"
%%% End:
