%---------------------------------------------------------------------
%
%                      resumen.tex
%
%---------------------------------------------------------------------
%
% Contiene el capítulo del resumen.En Inglés
%
% Se crea como un capítulo sin numeración.
%
%---------------------------------------------------------------------

\chapter{Summary}
\cabeceraEspecial{Summary}

%\begin{FraseCelebre}
%\begin{Frase}

%\end{Frase}
%\begin{Fuente}

%\end{Fuente}
%\end{FraseCelebre}

High throughput Proteomics, also called Shotgun Proteomics,
based on protein separation through 
electrophoresis and peptide separation through chromatography, followed
by tandem mass spectrometry, is a core technology for modern Proteomics.
In Proteomics experiments, following the experimental procedures that
lead to spectra acquisition, results are obtained when peptides are 
identified and the originating proteins present in the samples are inferred.
In order to do that, search engines use different strategies and then
results are statistically assessed through methods such as the False
Discovery Rate, FDR or other more complex statistical models.

The presence of proteomic data related to the fungal opportunistic 
pathogen \textit{Candida albicans} on online public repositories was
until recently very sparse, sometimes originated from low resolution
instruments and therfore not very robust.

In this context, the development databases and the adoption
of standard formats in Proteomics have an essential role in the analysis,
sharing and dissemination of results. The software tools presented here 
contribute to these objectives.

The database and web tool Proteopathogen was the first described software 
that combined Proteomics experiments results with specific information 
relevant to the study of \textit{C.albicans} proteins such as GO (Gene Ontology)
terms and KEGG (Kyoto Encyclopedia of Genes and Genomes) pathways annotations.
Following a first version where the results were collected
in a tab separated text format dependant on the software used to generate 
them (Vialas2009b), the software was completely refurbished to adapt to
the standard format for protein and peptide identifications, mzIdentML 
\citep{Vialas2015}. Proteopathogen is up to date an online tool to allow
visualization and analysis of Proteomics experiments results by the wet-lab 
users but also enabling revision by reviewers of the journals of the field.
\newpage
In parallel, the development of a \textit{Candida albicans} PeptideAtlas
means the most exhaustive proteome characterization up to the current date.
It describes over 71000 detected peptides assigned to 4174 proteins, which
represent 66\% of the predicted proteome. Is is, in addition, the first 
model of a fungal pathogen present in the global PeptideAtlas project and
benefits from the robustness and confidence that TPP (Trans Proteomics Pipeline)
enables.

\endinput
% Variable local para emacs, para  que encuentre el fichero maestro de
% compilación y funcionen mejor algunas teclas rápidas de AucTeX
%%%
%%% Local Variables:
%%% mode: latex
%%% TeX-master: "../Tesis.tex"
%%% End:
